test/e2e/testing.go

```diff
@@ -37,6 +57,22 @@ func TestAuthzServerWithWithOpts(module string, path string, addr string, opts .
 		return nil, err
 	}
 	m.Register(plugin.PluginName, internal.New(m, cfg))
+
+	//services := []string{"s1", "s3"}
+
+	m.Register("test_plugin", &testPlugin{})
```

Comment:
Can you please help me understand why we're registering this and how we're using it?

Reply:
I needed to enable decision_logs .  

Looking at https://github.com/open-policy-agent/opa-envoy-plugin/blob/064d6406c230b16ff17fd8c03500b42a89ec5914/envoyauth/evaluation_test.go#L212 , I figured that this is one way we could do it. 

Please suggest if there is any better way.